templates/getting-started/x-app.html

```diff
@@ -33,21 +30,21 @@
         color: #fff;
       }
 
-      app-header paper-icon-button {
+      paper-icon-button {
         --paper-icon-button-ink-color: white;
       }
 
     </style>
 
     <app-drawer-layout>
 
-      <app-drawer>
+      <app-drawer slot="drawer">
         <app-toolbar>Getting Started</app-toolbar>
       </app-drawer>
 
       <app-header-layout>
 
-        <app-header reveals effects="waterfall">
+        <app-header slot="header" reveals effects="waterfall">
           <app-toolbar>
             <paper-icon-button icon="menu" drawer-toggle></paper-icon-button>
```

Comment:
`drawer-toggle` attribute no longer automatically set the element to “display: none” in narrow layout. You can do something like in the app-drawer-layout demo to hide it: https://github.com/PolymerElements/app-layout/blob/2.0-preview/app-drawer-layout/demo/index.html#L66